src/redux/thunk.js

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'http://localhost:3000/messages/random';
+
```

Comment:
- [ ] I am not able to get greetings on the `frontend` that is probably because you did not setup `cors` on the backend please fix the issue
<img width="697" alt="Screenshot 2023-09-19 at 20 07 41" src="https://github.com/unorjikingsley/hello-react-front-end/assets/44745855/2bd14ba1-7fe8-47de-b1d9-e109a01333e0">

Reply:
thank you for the keen observation. I have made the corrections.